compiler/safetylib/safetyInterpreter.ml

```diff
@@ -409,6 +409,7 @@ let split_div sg ws es =
 let safe_opn safe opn es =
   let id =
     Sopn.get_instr_desc
+      X86_decl.x86_decl.reg_size
```

Comment:
For the moment, I have not be able to understand why we have an extra reg_size

Reply:
Exactly. The type of of this operator is `Uptr → ()`. But this might need to change: there sure is a better design awaiting for us to find it!